code/lib/types/src/modules/story.ts

```diff
@@ -46,6 +46,8 @@ export type ProjectAnnotations<TRenderer extends Renderer> = CsfProjectAnnotatio
 
   /* @deprecated use renderToCanvas */
   renderToDOM?: RenderToCanvas<TRenderer>;
+
+  beforeAll?: () => MaybePromise<void | (() => MaybePromise<void>)>;
```

Comment:
move to CSF

Reply:
👉 https://github.com/ComponentDriven/csf/pull/96